.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+name: Format and Compile
+
+on:
+  pull_request:
+    branches: [develop]
+
+jobs:
+  build-linux:
+    name: Format and Compile
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Clone falcon-core repository
+        run: |
+          cd ..
+          git clone --depth 1 https://github.com/falcon-eyrie/falcon-core.git
+          cd falcon-core
+          git checkout 06c07c1218a29d1420867556317b543b98d2a28b
+          cd ..
```

Comment:
I have problem with the pipeline having a fix commit. 

How you will handle the master branch to run with the falcon core master branch and the develop branch to run with the falcon core develop branch ?

Reply:
Can you automatize it with the github.base_ref variable ?